application-gateway-logviewer-goaccess/azuredeploy.parameters.json

```diff
@@ -3,19 +3,28 @@
   "contentVersion": "1.0.0.0",
   "parameters": {  
     "adminUsername": {
-      "value": "ubuntu"
+      "value": "GEN-UNIQUE"
     },
     "adminPassword": {
       "value": "GEN-PASSWORD"
     },
     "dnsNameForPublicIP": {
       "value": "GEN-UNIQUE-12"
+    },
+    "vmSize": {
+      "value": "Standard_A2_v2"
     },
 	  "appGwAccessLogsBlobSasUri": {
 	    "value" : "GEN-UNIQUE-24"
     },
-    "appGwAccessLogsBlobUrlIncludeRegex": {
+    "filterRegexForAppGwAccessLogs": {
 	    "value" : ".*"
-	  }
+    },
+    "_artifactsLocation": {
```

Comment:
remove this, you must use the defaultValue in the template and pull artifacts from the repo, not a private location